spec/lib/formatter/csv/subject_spec.rb

```diff
@@ -93,7 +93,7 @@
 
     context "with a subject that has no location metadata" do
       it "should match the db ordered subject_locations array" do
-        allow_any_instance_of(Medium::ActiveRecord_Associations_CollectionProxy)
+        allow_any_instance_of(Medium.const_get('ActiveRecord_Associations_CollectionProxy'))
```

Comment:
this seems strange that you have to access the collection proxy through a const get lookup... do you have any details on this?

Reply:
I think it has to do with this https://github.com/rails/rails/blob/v5.1.7/activerecord/lib/active_record/relation/delegation.rb#L8-L24
we cannot access `ActiveRecord_Associations_CollectionProxy` in the way that we did

Reply:
can you use this syntax `Medium::CollectionProxy` instead of the `const_get` one? It seems to resolve to the same AR proxy class.